addons/mail/static/src/xml/activity_view.xml

```diff
@@ -25,10 +25,11 @@
                         </div>
                     </span>
                 </div>
-                <KanbanColumnProgressBarAdapter Component="widgetComponents.KanbanColumnProgressBar"
-                    widgetArgs="[getProgressBarOptions(type[0]), getProgressBarColumnState(type[0])]"
-                    t-if="activityTypeIds.includes(type[0])"
-                    t-on-set-progress-bar-state="_onSetProgressBarState"/>
+                <div t-if="activityTypeIds.includes(type[0])">
```

Comment:
Why not just `<t>` instead of `<div>`?

Reply:
@alexkuhn yes can't use `<t>` because here we have `<div>` with `t-else` just below that ([see](https://github.com/odoo/odoo/pull/80944/files#diff-a95f555279057ac86c203db0bbec4f2e1e12a9a215b05302af205c49000e1c3bL32))

Reply:
ok